Day00 v1.1 (#55)
42-AI/bootcamp_data-engineering: day00/ex01/ex01.md

4) Remove the `ID` duplicates.

5) Convert the data type of the columns `Age Rating`, `User Rating Count` and `Size` to int.

> 6) Remove the rows with a `Name` length below 4 characters. 

Reviewer: ```suggestion
6) Remove the rows whose `Name` length is lower than 4 characters. 
```
